dynamo/prediction/fate.py

```diff
@@ -27,8 +27,8 @@ def fate(
     init_states: Optional[np.ndarray] = None,
     basis: Optional[None] = None,
     layer: str = "X",
-    dims: Union[tuple([list, None] + list(np.ScalarType))] = None,
-    genes: Union[list, None] = None,
+    dims: Optional[Union[tuple([List[Union[np.ScalarType]]] + list(np.ScalarType))]] = None,
```

Comment:
The typing suggested by ChatGPT will raise an error because Union requires each arg to be a type and np.ScalarType is not a type. Do you mean dims can only be an integer and a list of integers?